src/cheri_insts.sail

```diff
@@ -189,6 +189,19 @@ function clause execute (CSpecialRW(cd, scr, cs1)) =
 }
 
 union clause ast = CAndPerm : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is replaced with the contents of capability
+ * register *cs1* with the **perms** field set to the bitwise and of its
+ * previous value and bits 0 to [cap_hperms_width]-1 of integer register *rs2*
+ * and the **uperms** field set to the bitwise and of its previous value and
+ * bits [cap_uperms_shift] to [cap_uperms_shift]+[cap_uperms_width]-1 of *rd*.
+ *
+ *
+ * ## Exceptions:
+ * An exception is raised if:
+ *   - *cs1.tag* is not set.
```

Comment:
In two minds about whether this should be:
```suggestion
 *   - *cs1*.tag is not set.
```
or
```suggestion
 *   - *cs1*.**tag** is not set.
```
instead of what's there. (On the grounds that `cs1` is specifically referencing an operand named above, whereas `tag` is a property)

Reply:
`*cs1*.**tag** is not set.` matches MIPS, so I'll do that.

Reply:
(the latter being the style currently used for MIPS)